web/yo/app/scripts/controllers/gene.js

```diff
@@ -3589,6 +3599,7 @@ angular.module('oncokbApp')
                         'Sorry for any inconvinience.');
                 }
                 $scope.fileEditable = false;
+                $scope.geneEditable = $scope.fileEditable;
```

Comment:
I was thinking about that. Should I? Since we have been trying to reduce watcher usage.